tools/docker/BUILD

```diff
@@ -15,10 +15,39 @@
 ################################################################################
 #
 load("@io_bazel_rules_docker//container:image.bzl", "container_image")
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_push",
+)
 
 container_image(
     name = "envoy",
     base = "@distroless_cc//image",
     tags = ["manual"],
     tars = ["//src/envoy:envoy_tar"],
 )
+
+container_image(
+    name = "envoy_bionic",
+    base = "@bionic//image",
+    tags = ["manual"],
+    tars = ["//src/envoy:envoy_tar"],
+)
+
+container_push(
+    name = "push_envoy",
+    format = "Docker",
+    image = ":envoy",
+    registry = "gcr.io",
+    repository = "istio-testing/envoy",
+    tag = "{BUILD_CONFIG}-{BUILD_SCM_REVISION}",
+)
+
+container_push(
+    name = "push_envoy_bionic",
+    format = "Docker",
+    image = ":envoy_bionic",
+    registry = "gcr.io",
+    repository = "istio-testing/envoy_bionic",
```

Comment:
Is there a way to not hardcode target repository? We need to make this configurable for GKE builds.

Reply:
Yes, we can add build variables. Let me see how to generalize this.